sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -1,8 +1,16 @@
-# Azure Event Hubs Checkpoint Store for Azure Storage blobs client library for .NET
+# Azure Event Hubs Event Processor client library for .NET
 
-Intended as a companion to the `Azure.Messaging.EventHubs` client library, the Azure Event Hubs Checkpoint Store for Azure Storage Blobs enables using an Azure Storage account as the durable persistence mechanism for an `EventProcessor`.  The constructs in this library plug into the `EventProcessor` allowing it to preserve its state, in the form of checkpoints, as Azure storage blobs.
+Azure Event Hubs is a highly scalable publish-subscribe service that can ingest millions of events per second and stream them to multiple consumers. This lets you process and analyze the massive amounts of data produced by your connected devices and applications. Once Event Hubs has collected the data, you can retrieve, transform and store it by using any real-time analytics provider or with batching/storage adapters.  If you would like to know more about Azure Event Hubs, you may wish to review: [What is Event Hubs](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-about)?
 
-[Source code](.) | [Package (NuGet)](https://www.nuget.org/packages/Azure.Messaging.EventHubs.CheckpointStore.Blob/) | [API reference documentation](https://azure.github.io/azure-sdk-for-net/eventhub.html) | [Product documentation](https://docs.microsoft.com/en-us/azure/event-hubs/)
+The Event Processor client library is a companion to the Azure Event Hubs client library, providing a stand-alone client for consuming events in a robust, durable, and scalable way that is suitable for the majority of production scenarios.  An opinionated implementation built using Azure Storage Blobs, the Event Processor is recommended for:
+
+- Reading and processing events across all partitons of an Event Hub at scale with resilience to transient failures and intermittent network issues.
+
+- Procesing events cooperatively, where multiple processors dynamically distribute and share the responsibility in the context of a consumer group, gracefully managing the load as processors are added and removed from the group.
+
+- Managing checkpoints and state for processing in a durable manner using Azure Storage blobs as the underlying data store.
```

Comment:
Should we stick to `Azure Storage blobs` or to `Azure Storage Blobs`? It seems this file is not consistent with that.

Reply:
It depends on whether we're talking about the service (Blobs)  or the unit of storage that things are held in (blobs).     Their branding is.... weird.

Reply:
I did a pass and it looks like we could get away with consistently talking about the things that we hold data in.   Normalized to `blobs`